src/figuresClasses.ts

```diff
@@ -1,11 +1,104 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  private sideA: number;
+
+  private sideB: number;
+
+  private sideC: number;
```

Comment:
Same here. There is the same logic, you don't need to separate them. Fix this in other places as well pls